xchange-bitstamp/src/main/java/info/bitrich/xchangestream/bitstamp/v2/BitstampStreamingMarketDataService.java

```diff
@@ -0,0 +1,77 @@
+package info.bitrich.xchangestream.bitstamp.v2;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import info.bitrich.xchangestream.bitstamp.dto.BitstampOrderBook;
+import info.bitrich.xchangestream.bitstamp.dto.BitstampWebSocketTransaction;
+import info.bitrich.xchangestream.core.StreamingMarketDataService;
+import info.bitrich.xchangestream.service.netty.StreamingObjectMapperHelper;
+import io.reactivex.Observable;
+import org.knowm.xchange.bitstamp.BitstampAdapters;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.marketdata.OrderBook;
+import org.knowm.xchange.dto.marketdata.Ticker;
+import org.knowm.xchange.dto.marketdata.Trade;
+import org.knowm.xchange.exceptions.NotAvailableFromExchangeException;
+
+import java.util.Date;
+
+/**
+ * Bitstamp WebSocket V2 Streaming Market Data Service implementation
+ * Created by Pavel Chertalev on 15.03.2018.
+ */
+public class BitstampStreamingMarketDataService implements StreamingMarketDataService {
+    private final BitstampStreamingService service;
+
+    public BitstampStreamingMarketDataService(BitstampStreamingService service) {
+        this.service = service;
+    }
+
+    public Observable<OrderBook> getDifferentialOrderBook(CurrencyPair currencyPair, Object... args) {
```

Comment:
Maybe `getFullOrderBook()` or `getL3OrderBook()` might be better names here?  "Diff" order book didn't mean much to me until I looked at Bitstamp's API docs where it explains it's the full order book channel.

Reply:
I agree. It has name "Live full order book" on Bitstamp Api page. But 
BitstampStreamingMarketDataService v1 has the same method. Maybe be it will be better to keep absolutely the same signature like v1. What do u think?

Reply:
It's a new API. I think you're free to give it whatever name is best. No real benefit to keeping the old name.

Reply:
method has been renamed

Reply:
Typo `getfullOrderBook` -> `getFullOrderBook`

Reply:
Oh sorry! I have renamed v1 and with typo) Just commited fixed renaming